src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,68 @@
+package duke.task;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+
+    /**
+     * Creates a task.
+     *
+     * @param description the content of the class.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+
+    public Task(String description, boolean isDone) {
+        this.description = description;
+        this.isDone = isDone;
+    }
+
+
+    protected String getStatusIcon() {
```

Comment:
Perhaps it may be better to use the /** */ format for writing JavaDoc for this method instead of having it within the method itself?